ts/model_service_worker.py

```diff
@@ -100,6 +110,12 @@ def load_model(load_model_request):
             if "limitMaxImagePixels" in load_model_request:
                 limit_max_image_pixels = bool(load_model_request["limitMaxImagePixels"])
 
+            metrics = MetricsCacheYaml(
```

Comment:
Since we are creating an instance of `MetricsCacheYaml` here which is stored in context at the time of model loading and we also now have a mechanism to reset metrics after we emit them, I believe we should not be creating instances of `MetricsStore` objects when `predict` is called in `service.py` and replacing the `MetricsCacheYaml` object with the `MetricsStore` object?
https://github.com/pytorch/serve/pull/1727/files#diff-46d0c43f520da01cd15f2b4784a1545eafb2b2c6a8237088f6d79f641b75e19cL95-R114

```
metrics = MetricsStore(req_id_map, self.context.model_name)
self.context.metrics = metrics
```

If this is the case then it should be safe to remove `metrics_store.py` since it is no longer used anywhere?